bootstrap/src/main/java/org/jboss/windup/bootstrap/Bootstrap.java

```diff
@@ -163,7 +168,7 @@ else if ("--batchMode".equals(arg) || "-b".equals(arg))
             }
             else if ("--evaluate".equals(arg) || "-e".equals(arg))
             {
-                isEvaluate = true;
+                System.out.println("\"" + arg + "\" is no longer required!");
```

Comment:
I believe that it does. If there is no command and no windup options, it is supposed to print a help message. I think it even works. :)